src/goto-programs/interpreter_class.h

```diff
@@ -24,14 +24,14 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "goto_trace.h"
 #include "json_goto_trace.h"
 
-class interpretert:public messaget
+class interpretert
 {
 public:
   interpretert(
     const symbol_tablet &_symbol_table,
     const goto_functionst &_goto_functions,
     message_handlert &_message_handler)
-    : messaget(_message_handler),
+    : output(_message_handler),
```

Comment:
Why is this called `output` rather than `log` here?

Reply:
The interpreter necessarily provides user-facing output, i.e., is not useful if not printing some of the messages. So I thought "output" may be a better choice than "log" on this occasion. But I might well be wrong?